src/sentry/auth/helper.py

```diff
@@ -628,6 +628,7 @@ def _handle_existing_identity(self, auth_identity, identity):
             return HttpResponseRedirect(auth.get_login_redirect(self.request))
 
         self.clear_session()
+        metrics.incr('login-sso.success', tags={'provider': self.provider.provider})
```

Comment:
@mattrobenolt: Were you also suggesting tracking auth failures?

That does get a bit trickier since there's quite a few different points of failure.

Reply:
Ideally, yeah.